frontend/test/__support__/cypress.js

```diff
@@ -267,6 +267,17 @@ export function expectedRouteCalls({ route_alias, calls } = {}) {
   });
 }
 
+export function remapDisplayValueToFK(display_value, fk) {
+  // Both display_value and fk are expected to be field IDs
+  // You can get them from frontend/test/__support__/cypress_sample_dataset.json
+  cy.request("POST", `/api/field/${display_value}/dimension`, {
+    field_id: display_value,
+    name: "Product ID",
```

Comment:
This cannot be a static name. Either add a parameter to the function, or if possible, get the name dynamically.

Reply:
@flamber Thanks. 
Getting the name dynamically would complicate things.. A LOT. Providing `name` as an argument is probably our best option here.

Please see https://github.com/metabase/metabase/pull/14736#discussion_r573883912.